### ISSUE TITLE
[FEATURE REQUEST] save Customer Reviews settings

### ISSUE THREAD

**Issue by bottleblue** [IsThereAnyDeal/AugmentedSteam#2031]
### Current Behavior (if any)
Steam doesn't remember or consistently remember Customer Reviews settings across different store pages.

### Desired Behavior
To have Steam remember your Customer Reviews settings. Especially since Steam added a new option "Use new helpfulness system". I have to disable this every time I read reviews for each game. 

### Motivation / Use Case for Changing the Behavior
It's annoying to have to change settings for each new game page when I read reviews. 

Firefox doesn't remember any Customer Review settings across different store pages. If I configure the settings and then refresh the page it remembers REVIEW TYPE, PURCHASE TYPE, LANGUAGE, PLAYTIME, DISPLAY order (Summary, Most Helpful, Recent, Funny) and Use new helpfulness system.

Brave remembers LANGUAGE, PLAYTIME, DISPLAY order (Summary, Most Helpful, Recent, Funny) across different store pages. If I configure the settings and then refresh the page it remembers PURCHASE TYPE, LANGUAGE, PLAYTIME, DISPLAY order (Summary, Most Helpful, Recent, Funny) but not Use new helpfulness system..

**Comment by candela97** [comment]
We have such a feature, see #1313 and linked PR. Unfortuntely Steam changed some stuff and it's broken now, so you'll have to wait for the 4.1 update to pass store review.

I have a feeling the "Use new helpfulness system" opt-out will be removed at some point, so I'm not sure whether we should support that...

**Comment by bottleblue** [comment]
How do you make it save the settings? On Brave, AS is updated to 4.10 and it doesn't remember the review settings across different pages.

**Comment by candela97** [comment]
Works on my end 🤔

**Comment by bottleblue** [comment]
I said some of the settings are saved in Brave but not all. What should be selected for storage location in AS option? Local database or synced/local storage?

**Comment by candela97** [comment]
> I said some of the settings are saved in Brave but not all.

Yes, I mentioned that in my reply

> I have a feeling the "Use new helpfulness system" opt-out will be removed at some point, so I'm not sure whether we should support that...


The storage location has nothing to do with this feature.

**Comment by bottleblue** [comment]
You said it works on your end. Does every setting get saved for you?

**Comment by candela97** [comment]
Yes, the 4 types we support should work. See linked PR in the referenced issue for the rationale.

**Comment by bottleblue** [comment]
I guess it's mostly working on Brave. I can never get Purchase Type to save though.

**Comment by candela97** [comment]
It's saved per-app. I guess we can modify the cookie path and make the expiry longer 🤔

**Comment by bottleblue** [comment]
Not sure if it's my settings or what. I have ublock disabled on steam and cookies whitelisted. Oh well it's not a huge deal. Thx for the help!
